pages/cats.ts

```diff
@@ -1,35 +1,37 @@
-import { Locator, Page, expect } from "@playwright/test";
-
-export class CatsPage {
-  readonly page: Page;
-  readonly navigationBar: string;
-  readonly header: Locator;
-
-  readonly addCatButton: Locator;
-  readonly removeCaButton: Locator;
-  readonly apocalypseButton: Locator;
-
-  readonly catCards: Locator;
-
-  constructor(page: Page) {
-    this.page = page;
-    this.navigationBar = "nav";
-    this.header = page.getByRole("heading", { name: "Tlačítka" });
-
-    // OH! Fill this! Check the HTML and fill the selectors. Use whatever selector you want.
-    this.addCatButton;
-    this.removeCaButton;
-    this.apocalypseButton;
-    this.catCards;
-  }
-
-  async visit() {
-    await this.page.goto("/adding.html");
-  }
-
-  // or you can use expext(pages.catPage.catCards).toHaveCount(3)
-  // or different number
-  getCountOfCatCards = async () => {
-    return await this.catCards.count();
-  };
-}
+import { Locator, Page, expect } from "@playwright/test";
+
+export class CatsPage {
+  readonly page: Page;
+  readonly navigationBar: string;
+  readonly header: Locator;
+
+  readonly addCatButton: Locator;
+  readonly removeCatButton: Locator;
+  readonly apocalypseButton: Locator;
+
+  readonly catCards: Locator;
+  readonly counter: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.navigationBar = "nav";
+    this.header = page.getByRole("heading", { name: "Tlačítka" });
+
+    // OH! Fill this! Check the HTML and fill the selectors. Use whatever selector you want.
+    this.addCatButton = page.locator("#addItem");
+    this.removeCatButton = page.locator("#removeItem");
+    this.apocalypseButton = page.locator("#removeAll");
+    this.catCards = page.getByAltText("Kočka");

```

Comment:
Asi bych našel lepší lokátor, což? tohle má být celá kartička, ne jen obrázek. Takhle Ti to najde obrázek, který má popis "Kočka", což taky funguje, protože jakmile se zobrazí obrázek kočky, tak se zobrazí i kartička.. Ale pokud tohle funguje, tak jsem s tím OK, jen bych viděl lepší možnost.